src/main/resources/api.yaml

```diff
@@ -340,7 +378,7 @@ components:
           type: "string"
           maxLength: 256
         verified:
-          type: "boolean"
+          type: "integer"
```

Comment:
Why it's an integer?

Reply:
I changed it due to `emial_status TINYINT` in the Spec for DB. I thought the corresponding one will be `byte` or `int` in Java.

Reply:
That's just for extendibility. In the external interface, `verified` should still be boolean anyway.

Reply:
Revert to `boolean` now.